src/Polly.Core/Simmy/Outcomes/ChaosOutcomeStrategyOptions.cs

```diff
@@ -28,5 +28,5 @@ public class ChaosOutcomeStrategyOptions<TResult> : ChaosStrategyOptions
     /// Defaults to <see langword="null"/>. This property is required.
     /// </value>
     [Required]
```

Comment:
Should it still be required if we allow null?

I'm genuinely confused at this point as to what problem we're solving by changing this now 😅

Reply:
It doesn't have a default value so it's `null` by default because that's the only value we can assign to it. 

It's the same pattern that we used for fallback:

https://github.com/App-vNext/Polly/blob/41d27ff055df2175d6b048a823d3131158ff20b1/src/Polly.Core/Fallback/FallbackStrategyOptions.TResult.cs#L32